primefaces/src/main/java/org/primefaces/renderkit/MenuItemAwareRenderer.java

```diff
@@ -114,7 +114,7 @@ protected void encodeOnClick(FacesContext context, UIComponent source, MenuItem
 
             if (isLink) {
                 // allow CTRL+CLICK link to open new tab
-                command = "if(event.ctrlKey||event.metaKey){return true};" + command;
+                command = "if(PrimeFaces.utils.isMetaKey(event)){return true};" + command;
```

Comment:
It's not possible to have this as `PF.isMetaKey` (which is way shorter)?

Reply:
PF is only for widgets so the only shorter way would be `PrimeFaces` only but then this is a utility and belongs in the utils namespace

```js
    PF = function(widgetVar) {
    	var widgetInstance = PrimeFaces.widgets[widgetVar];

    	if (!widgetInstance) {
	        PrimeFaces.widgetNotAvailable(widgetVar);
    	}

        return widgetInstance;
    };
```

Reply:
OK check out my latest commit.

Reply:
> PF is only for widgets ...

I don't know what made me think it was an object. 😅